djstripe/models/base.py

```diff
@@ -562,7 +562,7 @@ def sync_from_stripe_data(cls, data, field_name="id"):
 			data, field_name=field_name, current_ids=current_ids
 		)
 
-		if not created:
+		if instance and not created:
```

Comment:
Wait, instance can be none?

Reply:
ok I see the context from the linked issue. uuuurgh, that feels extremely wrong. the PR is gtm though

Reply:
I think particular bug's only occurring because my implementation of `InvoiceItem.sync_from_stripe_data`, so another fix would be to to have that check for `invoice` being set before calling `Invoice.sync_from_stripe_data`.

https://github.com/dj-stripe/dj-stripe/blob/9693f21742b562396a8ddfb73feea56dfc30b46f/djstripe/models/billing.py#L623

But maybe this PR's way is safer since `_get_or_create_from_stripe_object` can return `None, False`

https://github.com/dj-stripe/dj-stripe/blob/9693f21742b562396a8ddfb73feea56dfc30b46f/djstripe/models/base.py

Reply:
that function shouldn't ever return None tbh.